Makefile.am

```diff
@@ -63,11 +63,64 @@ AM_CPPFLAGS = \
     $(ICU_CFLAGS)
 
 if HAVE_LDAP
-AM_CPPFLAGS += $(LDAP_CPPFLAGS)
+    AM_CPPFLAGS += $(LDAP_CPPFLAGS)
```

Comment:
There is no consensus on indenting if-blocks and many other if-blocks start at the first column.  So I am against introducing indentation for a line, without changing it anyhow.